### PR TITLE
Next

### PR DESCRIPTION
**Pull request by fraternl**
few cosmetic changes

### CODE REVIEW

File: radio/src/translations/nl.h.txt [hunk 1]
```diff
@@ -656,11 +656,11 @@
 #define TR_ELIMITS             TR("E.Limits", "Extended Limits")
 #define TR_ETRIMS              TR("E.Trims", "Extended Trims")
 #define TR_TRIMINC             "Trim Step"
-#define TR_DISPLAY_TRIMS       TR("Show Trims", "Display Trims")
+#define TR_DISPLAY_TRIMS       TR("Toon Trims", "Toon Trims")
```
Comment (schwabe) [inline radio/src/translations/nl.h.txt]: if the strings are identical you can remove the TR

Comment (fraternl) [inline radio/src/translations/nl.h.txt]: Should I correct this before you merge?

Comment (kilrah) [inline radio/src/translations/nl.h.txt]: Yes

File: radio/src/translations/nl.h.txt [hunk 2]
```diff
@@ -1077,7 +1077,7 @@
 #define TR_EEBACKUP            "EEPROM backup"
 #define TR_FACTORYRESET        "Factory reset"
 #else
-#define TR_EEBACKUP            TR("\001[ENTER Lang]: Backup", "\012[ENTER Lang]: EEPROM backup naar SD-CARD")
+#define TR_EEBACKUP            TR("\001[ENTER Lang]: Backup", "\012[ENTER Lang]: EEPROM backup => SD")
```
Comment (kilrah) [inline radio/src/translations/nl.h.txt]: Yep it is, sorry

**Pull request merged**